app/models/concerns/image.rb

```diff
@@ -0,0 +1,22 @@
+module Image
```

Comment:
The name Image seems to be very generic, is it okay or should we rename it into something more specific? depending on what code we are planning to add inside. Based on your code here is a suggestion, can it be ImageValidator or something like that?

Reply:
Module name changed to ImageValidation